### PR TITLE
Cloud hosted adjustments

### PR DESCRIPTION
**Pull request by Kevcpro**

### CODE REVIEW

**Review by gkwan-ibm — CHANGES_REQUESTED** [review]
@Kevcpro 
- at step4, instead of using launch app, better use curl commands `curl -s http://localhost:9080/artists | jq`
- at step 10, need to tell users where to run `mvn generate-resources`. The static guide need too.
- at the summary step, Nice Work!, the statement is disjointed

**Comment by gkwan-ibm** [comment]
@Kevcpro 
- at step 10, need to tell users where to run `mvn generate-resources`. The static guide need too.
  - I think should be the `start` directory with backticks instead of root
- at the summary step, Nice Work!, the statement is disjointed
  - is not fixed yet

File: README.adoc [hunk 1]
```diff
@@ -80,7 +88,20 @@ include::{common-includes}/gitclone.adoc[]
...
 
+// Static guide instruction
+ifndef::cloud-hosted[]
 Then, point your browser to the web application root http://localhost:9080/app[http://localhost:9080/app^] to see the following output:
+endif::[]
+
+// Cloud hosted guide instruction
+ifdef::cloud-hosted[]
+Select **Launch Application** from the menu of the IDE, 
+type in **9080** to specify the port number for the microservice, and click the **OK** button.
```
Comment (dmuelle) [inline README.adoc]: type in-->type

File: README.adoc [hunk 2]
```diff
@@ -338,8 +371,19 @@ mvn generate-resources
...
+// Static guide instruction
+ifndef::cloud-hosted[]
 Point your browser to the http://localhost:9080/app[http://localhost:9080/app^] 
 web application root to see the following output:
+endif::[]
+
+// Cloud hosted guide instruction
+ifdef::cloud-hosted[]
+Select **Launch Application** from the menu of the IDE, 
+type in **9080** to specify the port number for the microservice, and click the **OK** button.
```
Comment (dmuelle) [inline README.adoc]: type in-->type

File: README.adoc [hunk 3]
```diff
@@ -362,9 +406,18 @@ A web browser runs JavaScript, and the `curl` command doesn't.
...
+// Static guide instruction
+ifndef::cloud-hosted[]
 Whenever you change and build your Angular implementation, the application root at
 http://localhost:9080/app[http://localhost:9080/app^] reflects the changes
 automatically. 
+endif::[]
+
+// Cloud hosted guide instruction
+ifdef::cloud-hosted[]
+Whenever you change and build your Angular implementation the changes will be reflected automatically
```
Comment (dmuelle) [inline README.adoc]: Whenever you change and build your Angular implementation the changes will be reflected automatically...
--->
Whenever you change and build your Angular implementation, the changes are automatically reflected...

**Review by dmuelle — CHANGES_REQUESTED** [review]
a couple minor things, looks good

**Review by dmuelle — APPROVED** [review]
LGTM thanks

**Comment by gkwan-ibm** [comment]
addressed https://github.com/OpenLiberty/guides-common/issues/732

**Review by gkwan-ibm — APPROVED** [review]

**Review by davidwyao — APPROVED** [review]

**Pull request merged**